monitor/services/src/test/java/org/datacleaner/monitor/configuration/ComponentStoreImplTest.java

```diff
@@ -47,7 +47,7 @@ private Repository getRepositoryMock() {
 
     private RepositoryFolder getRepositoryFolderMock() {
         RepositoryFolder repositoryFolder = createNiceMock(RepositoryFolder.class);
-        expect(repositoryFolder.getFolder(ComponentStoreImpl.FOLDER_NAME)).andReturn(getComponentsFolderMock()).anyTimes();
+        expect(repositoryFolder.getOrCreateFolder(ComponentStoreImpl.FOLDER_NAME)).andReturn(getComponentsFolderMock()).anyTimes();
```

Comment:
That's not a change of this PR, but my 5c on the topic would be: Probably it doesn't matter but in the eyes of the unittest it also doesn't matter, it's just a specification of behaviour for the mock rather than a requirement that it happens 0, 1 or 1000 times.